packages/wonder-blocks-search-field/src/components/__tests__/search-field.test.js

```diff
@@ -0,0 +1,181 @@
+// @flow
```

Comment:
I tried to cover all the test cases I could think of for now, but it's very possible that I missed some. Are there any other tests I should add?